designs/virtualization/4.5/Scheduling/Node-selector/node-selector.md

```diff
@@ -0,0 +1,37 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Node Selector
+
+Introducing Node-selector as it appears in-page and the Edit modal.
+
+![Node selector field in the Scheduling section ](img/Node-selector-0-0.jpg)
+The node-selector field is pulled from the Workload details section and placed under the Scheduling section.
+
+In this example there is no node-selector set, meaning, no node labels added.
+Clicking on the edit (pencil) button will open the Edit modal.
+Clicking on the "O labels" link text will open that modal as well.
+
+
+![Node selector modal ](img/Node-selector-1-0.jpg)
+The modal consists of:
+- key=value pair text fields followed by a Remove button
+- Button to add another pair, or label
+- External link to the nodes list
+- "Nodes Checker", active notification that provides feedback if nodes found for the labels above
+
+When there is only one row of key=value pair, the Remove action will be disabled
+
+![Node selector modal ](img/Node-selector-1-1.jpg)
+On typing some value within any of the fields, the Node Checker will check for nodes with matching labels, with a time out of [1] second to kick in.
+
+![Node selector modal ](img/Node-selector-1-2.jpg)
```

Comment:
Nit or maybe an FYI: We're probably not super consistent with this elsewhere in the Console (maybe we should all agree on a convention) but at least in Overviews/Dashboards we standardized on using the word "View" for things that will take the user somewhere else (like "View details"). Related to that, maybe "Explore nodes list" should be "View nodes list".

"See" might be a better word to reveal the popover. "Show" is also possible, but at least OperatorHub uses that word to indicate that more things will show up inline when clicked.